src/main/java/org/sonarlint/intellij/core/SonarLintEngineManager.java

```diff
@@ -120,7 +121,7 @@ public synchronized StandaloneSonarLintEngine getStandaloneEngine() {
     return standalone;
   }
 
-  public synchronized ConnectedSonarLintEngine getConnectedEngine(SonarLintProjectNotifications notifications, String serverId, String projectKey) {
+  public synchronized ConnectedSonarLintEngine getConnectedEngine(SonarLintProjectNotifications notifications, String serverId, String projectKey) throws InvalidBindingException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'org.sonarlint.intellij.exception.InvalidBindingException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)